README.md

```diff
@@ -53,6 +53,8 @@ The following table lists the configurable parameters of the `kubernetes-externa
 | Parameter                                 | Description                                                  | Default                                                 |
 | ----------------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------- |
 | `env.AWS_REGION`                          | Set AWS_REGION in Deployment Pod                             | `us-west-2`                                             |
+| `env.LOG_LEVEL`                           | Set the application log level                                | `info`                                                  |
```

Comment:
Can you also update the charts README and values.yaml file with those two environment variables https://github.com/godaddy/kubernetes-external-secrets/tree/master/charts/kubernetes-external-secrets?